update_compilers/install_binaries.sh

```diff
@@ -8,32 +8,25 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 #########################
 # pahole
-
-TARGET_PAHOLE_VERSION=v1.19
-CURRENT_PAHOLE_VERSION=""
-if [[ -f "${OPT}/pahole/bin/pahole" ]]; then
-    CURRENT_PAHOLE_VERSION=$("${OPT}/pahole/bin/pahole" --version)
-fi
-
-if [[ "$TARGET_PAHOLE_VERSION" != "$CURRENT_PAHOLE_VERSION" ]]; then
+if [[ ! -f "${OPT}/pahole/bin/pahole" ]]; then
```

Comment:
so this will only ever install once, right? Is the intention to install "trunk" pahole? (like the other PR implies?)

Reply:
And by that I mean:
- elsewhere `trunk` not only means trunk, but the current (at least built daily, trunk)
- so I think we should mean the same here :)

This is OK as is but we should consider moving this to our yaml stuff wihch can deal with this much better.

Reply:
The intention is to point to master and build pahole everytime and install it in `${OPT}/pahole` (this dir is unchanged). Instead of having to upgrade versions every now and then.

Digging more made me realize that this script is invoked by a cronjob (every 3h). So I think the best way is to download and build the pahole and then install the binaries in the end to overwrite existing ones.

Reply:
This script is invoked daily at 3am, which is fin....we just need to remove the `if` check and it will install nightly.

In a separate PR we can make it build using our existing system. I'll make the change and merge in.